docs/features/logforwarding/outputs/splunk-forwarding.adoc

```diff
@@ -43,5 +49,113 @@ spec:
       outputRefs:
         - splunk-receiver
 ----
+1. `token` - Points to the secret containing the Splunk HEC token used for authenticating requests.
+2. `url` - The base URL of the Splunk instance.
+3. `index` - Optional. The name of the index to send events to. If not specified, the default index defined within Splunk is used. This supports template syntax to allow dynamic per-event values.
+4. `source` - Optional. The source of events sent to this sink. This supports template syntax to allow dynamic per-event values.
+5. `indexedFields` - Optional. Fields to be added to Splunk index.
+6. `compression` - Optional. Compression configuration, available to are: `none`, `gzip`. Default is `none`.
+
+=== `source`
+In Splunk, the `source` field typically identifies the origin of a log event.
+To ensure consistency and meaningful categorization, the `source` value can be dynamically derived from the `log_type` and `log_source` fields,
+following Cluster Log Forwarder's conventions.
+
+Below the table with default value depends on log_type and log_source will be used if not set in configuration.
```

Comment:
Let's expand this table to identify the 5?? splunk metadata fields and identify the values to which they will be set.